common/exception/src/exception_code.rs

```diff
@@ -118,6 +118,7 @@ build_exceptions! {
     PermissionDenied(1063),
     UnmarshalError(1064),
     SemanticError(1065),
+    InvalidAuthInfo(1066),
```

Comment:
what's the difference with AuthenticateFailure(1051)? q.q

Reply:
It is not used in the authentication process, exists for AuthInfo creation.

https://github.com/datafuselabs/databend/pull/4530/files#diff-50bc2112130914344462eb75dcd752843e2ca16edd6adcb3ba3a7b0589547f2eR45